frontend/public/locales/en/public.json

```diff
@@ -361,7 +361,7 @@
   "Not available": "Not available",
   "View settings": "View settings",
   "Cluster API address": "Cluster API address",
-  "Provider": "Provider",
+  "Provider": "Infrastructure Provider",
```

Comment:
lets also change the key 
```suggestion
  "Infrastructure provider": "Infrastructure provider",
```

Reply:
done

Reply:
FYI, I've begun work on https://issues.redhat.com/browse/CONSOLE-3124, which will migrate this test to Cypress.

Reply:
PR to migrate the test to Cypress:  https://github.com/openshift/console/pull/11332

Reply:
@rhamilto Thank you, updated